src/main/java/de/tum/in/www1/artemis/domain/Submission.java

```diff
@@ -53,6 +53,7 @@
     private SubmissionType type;
 
     @Column(name = "example_submission")
+    @JsonView(QuizView.Before.class)
```

Comment:
Can you add a JavaDoc comment why we need this?

Reply:
I think we actually do not need this - probably it slipped in, I removed it